tools/docker/docker-compose.integration.yaml

```diff
@@ -1,4 +1,4 @@
-version: '3.1'
+version: "3.1"
```

Comment:
Hm, the root workspaces `.prettierrc` should prevent this, we have single quotes enforced.

Reply:
hmmm weird does the `.prettierrc` apply to .yaml files? It might be the default styler on my editor. I thought it was weird in the .yaml files we had a mix of both single and double quotes - look at this file on develop.

Reply:
it does work on .yaml files